pkg/discovery/discovery.go

```diff
@@ -13,3 +14,10 @@ import (
 type Driver interface {
 	BroadcastPeer(addressee swarm.Address, overlay swarm.Address, addr ma.Multiaddr) error
 }
+
+// Peerer can suggest new known or connected peers to other peers
+type Peerer interface {
```

Comment:
@acud I am not sure this is the right place for this interface, please advise if wrong.

Reply:
looks good to me :+1: